src/utils/pii-remover.ts

```diff
@@ -72,17 +72,17 @@ export function stripPIIFromString(value: string) {
   stripped = stripped.replace(STATE_PATTERN, "state=[state]");
 
   if (isDate(value)) {
-    var DATE_REDACTION_STRING = "[date]";
+    let DATE_REDACTION_STRING = "[date]";
     stripped = stripped.replace(DATE_PATTERN_NUMERIC_1, DATE_REDACTION_STRING);
     stripped = stripped.replace(DATE_PATTERN_NUMERIC_2, DATE_REDACTION_STRING);
     stripped = stripped.replace(DATE_PATTERN_NUMERIC_3, DATE_REDACTION_STRING);
     stripped = stripped.replace(DATE_PATTERN_STRING_1, DATE_REDACTION_STRING);
     stripped = stripped.replace(DATE_PATTERN_STRING_2, DATE_REDACTION_STRING);
   } else if (isPostCode(value)) {
-    var POSTCODE_REDACTION_STRING = "[postcode]";
+    let POSTCODE_REDACTION_STRING = "[postcode]";
     stripped = stripped.replace(POSTCODE_PATTERN, POSTCODE_REDACTION_STRING);
   } else if (isPhoneNumber(value)) {
-    var PHONENUMBER_REDACTION_STRING = "[phonenumber]";
+    let PHONENUMBER_REDACTION_STRING = "[phonenumber]";
```

Comment:
because we will are not updating this variable after this line, if we have to change it's better to use CONST instead of let
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Reply:
ok changed